Controller/AdminSecurityController.php

```diff
@@ -64,12 +64,22 @@ public function loginAction()
             return new RedirectResponse($refererUri && $refererUri != $request->getUri() ? $refererUri : $this->container->get('router')->generate('sonata_admin_dashboard'));
         }
 
+        // TODO: Deprecated in 2.3, to be removed in 3.0
+        $resetRoute = 'sonata_user_admin_resetting_request';
+
+        // TODO: Deprecated in 2.3, to be removed in 3.0
+        if (null === $this->container->get('router')->getRouteCollection()->get('sonata_user_admin_resetting_request')) {
```

Comment:
Is that OK for you? Or should I provide a PR here as well? IMO a BcExtension would be better, because we can catch the exception right at generation time.